include/tvm/relay/pass.h

```diff
@@ -271,13 +272,17 @@ TVM_DLL tvm::Array<TypeVar> AllTypeVars(const Type& t, const Module& mod);
  */
 TVM_DLL Expr DeadCodeElimination(const Expr& e);
 
+transform::Pass DeadCodeEliminationPass();
```

Comment:
given that transform namespace has been merged in. Please move them to transform.h . Under the same name

```
Pass DeadCodeElimination();
```